terraform/modules/tf_threat_intel_downloader/dynamodb.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_dynamodb_table" "threat_intel_ioc" {
+  name           = "${var.table_name}"
+  read_capacity  = "${var.table_rcu}"
+  write_capacity = "${var.table_wcu}"
+  hash_key       = "value"
+  range_key      = "type"
+
+  attribute {
+    name = "value"
+    type = "S"
+  }
+
+  attribute {
+    name = "type"
+    type = "S"
+  }
+
+  ttl {
+    attribute_name = "expiration_date"
+    enabled        = true
+  }
+
+  tags {
+    Name = "ThreatIntel"
```

Comment:
should this be StreamAlert so it's picked up by billing?

Reply:
Hmmm, I will change it to `StreamAlert`. The reason I chose `ThreatIntel` as tag is because we may want to see how much does this standalone lambda function (Threat Intel Downloader) cost. Actually, we can easily get sense since this lambda function mainly use DynamoDB and it is the only feature using DynamoDB in StreamAlert.

Reply:
@chunyong-lin you can use multiple tags here if you'd like to keep threatintel as well